### PR TITLE
Update whitelist to allowlist

### PR DESCRIPTION
**Pull request by hbc-bindu**
### What does this PR do?

Update whitelist to allowlist per [Inclusive Content Initiative](https://salesforce.quip.com/fyWbAlOLjQap#MdLACAT8nll) 

### What issues does this PR fix or reference?
#<Insert GitHub Issue>, @<Insert GUS WI>@

### Functionality Before
<insert gif and/or summary>

### Functionality After
<insert gif and/or summary>

### CODE REVIEW

File: docs/_articles/en/apex/interactive-debugger.md [hunk 1]
```diff
@@ -76,9 +76,9 @@ To see your exception breakpoints, run **SFDX: Configure Apex Debug Exceptions**
 
 When you close VS Code, all your exception breakpoints are removed. (Your line breakpoints, however, remain.)
 
-## Whitelist Users and Request Types
+## Allowlist Users and Request Types
 
-To filter which requests are debugged, edit your `launch.json` file to set up whitelisting. (The `launch.json` file lives in your project’s `.vscode` directory.) If you don’t use whitelisting, all events in your org trigger debugging during a debugging session. Whitelist users or request types to focus only on the events that are relevant to the problem you’re debugging.
+To filter which requests are debugged, edit your `launch.json` file to set up allowed users list. (The `launch.json` file lives in your project’s `.vscode` directory.) If you don’t use an allowed users list, all events in your org trigger debugging during a debugging session. Set up allowlist users or request types to focus only on the events that are relevant to the problem you’re debugging.
```
Comment (ekapner) [inline docs/_articles/en/apex/interactive-debugger.md:81]: "an" allowed users list (?)

File: docs/_articles/en/apex/interactive-debugger.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ Add filters to the `"Launch Apex Debugger"` configuration:
 
 To auto-complete potential request type values for `"requestTypeFilter"`, press Ctrl+Space.
 
-To filter by entry point, enter a regular expression as the value for `"entryPointFilter"`. For example, to whitelist requests made by the Visualforce page `MyPage`, enter `".*/apex/MyPage.apexp"`.
+To filter by entry point, enter a regular expression as the value for `"entryPointFilter"`. For example, to allowlist requests made by the Visualforce page `MyPage`, enter `".*/apex/MyPage.apexp"`.
```
Comment (ekapner) [inline docs/_articles/en/apex/interactive-debugger.md:101]: Yes, this does sound funny now that I re-read it. I'm fine if you keep as-is. But will provide this suggestion: remove the "list" - how about? For example, to allow requests made by...

File: docs/_articles/en/apex/interactive-debugger.md (hunk 1 above)
Comment (hbc-bindu) [reply docs/_articles/en/apex/interactive-debugger.md]: done

File: docs/_articles/en/apex/interactive-debugger.md (hunk 2 above)
Comment (hbc-bindu) [reply docs/_articles/en/apex/interactive-debugger.md]: Done

**Pull request merged**